t/admin/balancer.t

```diff
@@ -25,7 +25,7 @@ add_block_preprocessor(sub {
 
     my $init_by_lua_block = <<_EOC_;
     require "resty.core"
-    apisix = require("apisix")
+    apisix = require("init")
```

Comment:
This is wrong, we should use the original name "apisix".

Reply:
`./?/init.lua;` is the standard Lua source code search path.

please take a look at this:

```diff
$ cat ../mm.patch
diff --git a/bin/apisix b/bin/apisix
index e27da893..c393536e 100755
--- a/bin/apisix
+++ b/bin/apisix
@@ -132,7 +132,7 @@ stream {

     init_by_lua_block {
         require "resty.core"
-        apisix = require("init")
+        apisix = require("apisix")
         apisix.stream_init()
     }

@@ -261,7 +261,7 @@ http {

     init_by_lua_block {
         require "resty.core"
-        apisix = require("init")
+        apisix = require("apisix")

         local dns_resolver = { {% for _, dns_addr in ipairs(dns_resolver or {}) do %} "{*dns_addr*}", {% end %} }
         local args = {
diff --git a/dashboard b/dashboard
index cfb3ee7b..5ce7c37c 160000
--- a/dashboard
+++ b/dashboard
@@ -1 +1 @@
-Subproject commit cfb3ee7b8721076975c1deaff3e52da3ea4a312a
+Subproject commit 5ce7c37c758f6365da042333d7909c12c0ab6714-dirty
diff --git a/t/APISIX.pm b/t/APISIX.pm
index e1a336a8..6b191524 100644
--- a/t/APISIX.pm
+++ b/t/APISIX.pm
@@ -107,7 +107,7 @@ _EOC_

     my $stream_enable = $block->stream_enable;
     my $stream_config = $block->stream_config // <<_EOC_;
-    lua_package_path "$apisix_home/deps/share/lua/5.1/?.lua;$apisix_home/apisix/?.lua;$apisix_home/t/?.lua;./?.lua;;";
+    lua_package_path "$apisix_home/deps/share/lua/5.1/?.lua;$apisix_home/apisix/?.lua;$apisix_home/t/?.lua;./?.lua;./?/init.lua;;";
     lua_package_cpath "$apisix_home/deps/lib/lua/5.1/?.so;$apisix_home/deps/lib64/lua/5.1/?.so;./?.so;;";

     lua_socket_log_errors off;
@@ -127,7 +127,7 @@ _EOC_

         require "resty.core"

-        apisix = require("init")
+        apisix = require("apisix")
         apisix.stream_init()
     }

@@ -177,7 +177,7 @@ _EOC_

     require "resty.core"

-    apisix = require("init")
+    apisix = require("apisix")
     local args = {
         dns_resolver = $dns_addrs_tbl_str,
     }
@@ -186,7 +186,7 @@ _EOC_

     my $http_config = $block->http_config // '';
     $http_config .= <<_EOC_;
-    lua_package_path "$apisix_home/deps/share/lua/5.1/?.lua;$apisix_home/apisix/?.lua;$apisix_home/t/?.lua;./?.lua;;";
+    lua_package_path "$apisix_home/deps/share/lua/5.1/?.lua;$apisix_home/apisix/?.lua;$apisix_home/t/?.lua;./?.lua;./?/init.lua;;";
     lua_package_cpath "$apisix_home/deps/lib/lua/5.1/?.so;$apisix_home/deps/lib64/lua/5.1/?.so;./?.so;;";

     lua_shared_dict plugin-limit-req     10m;
@@ -216,7 +216,7 @@ _EOC_
     }

     init_worker_by_lua_block {
-        require("init").http_init_worker()
+        require("apisix").http_init_worker()
     }

     # fake server, only for test
diff --git a/t/admin/balancer.t b/t/admin/balancer.t
index d88f19bf..0be70dbd 100644
--- a/t/admin/balancer.t
+++ b/t/admin/balancer.t
@@ -25,7 +25,7 @@ add_block_preprocessor(sub {

     my $init_by_lua_block = <<_EOC_;
     require "resty.core"
-    apisix = require("init")
+    apisix = require("apisix")
     apisix.http_init()

     function test(route, ctx, count)
diff --git a/t/admin/health-check.t b/t/admin/health-check.t
index 1020a30d..8680768e 100644
--- a/t/admin/health-check.t
+++ b/t/admin/health-check.t
@@ -27,7 +27,7 @@ add_block_preprocessor(sub {

     my $init_by_lua_block = <<_EOC_;
     require "resty.core"
-    apisix = require("init")
+    apisix = require("apisix")
     apisix.http_init()

     json = require("cjson.safe")
```